tree-sitter-mozcpp/src/scanner.cc

```diff
@@ -1 +0,0 @@
-../../tree-sitter-cpp/src/scanner.cc
```

Comment:
Why don't you keep a symbolic link here ?
Removing it means that we need to manually sync this file coming from tree-sitter-cpp.

Reply:
It didn't work on `Windows`, perhaps I could fix that using the `bash` shell instead of `Powershell`.

Reply:
good idea :) 
simple and easy (I was thinking about a more complex solution)

Reply:
Thanks to you for finding my initial horrible implementation! :)